tests/navigator/tincludefile.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: true # this test is flakey with testament all
+    disabled: true # this test is flakey with testament all
```

Comment:
Created around 2021-04 in a commit that references #17784
Disabled around 2021-07 due to flakiness.
probably should be removed.

Reply:
yeah... we're not going to bother with this navigator stuff, let's kill it.